crates/astria-sequencer/src/proposal/commitment.rs

```diff
@@ -27,7 +27,7 @@ impl GeneratedCommitments {
     /// which can be used as the block's transactions.
     #[must_use]
     pub(crate) fn into_transactions(self, mut tx_data: Vec<Bytes>) -> Vec<Bytes> {
-        let mut txs = Vec::with_capacity(tx_data.len() + 2);
```

Comment:
This is an optimization I think. I think it's reasonable to `tx_data.len().saturating_add(2)` here. If this ends up too small then the `Vec::push` calls below will panic (not that this will ever be a problem because we won't have enough memory...)

Reply:
Done in [44c99af](https://github.com/astriaorg/astria/pull/1081/commits/44c99affc52d66ee3134fa1bbc23f66d21bb632a).